src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -1088,3 +1088,17 @@ def update_local_user(cmd, client, resource_group_name, account_name, username,
                          home_directory, has_shared_key, has_ssh_key, has_ssh_password)
     return client.create_or_update(resource_group_name=resource_group_name, account_name=account_name,
                                    username=username, properties=local_user)
+
+
+def begin_failover(client, resource_group_name, account_name, failover_type=None, yes=None, **kwargs):
+    if not yes and failover_type and failover_type.lower() != "planned":
```

Comment:
If `failover_type` is `None`, we still need this Prompt, right?

Reply:
The default value of `failover_type` is `Planned`, so I don't think it needs Prompt.

Reply:
Is `None` for `failover_type` just means `Unplanned`? If so, I think you're right.